src/main/java/io/github/wimdeblauwe/hsbt/mvc/HtmxResponse.java

```diff
@@ -92,22 +185,41 @@ Set<String> getTemplates() {
 
     Map<String, String> getTriggers() {
         if(this.triggers.isEmpty()) {
+            //noinspection unchecked
             return Collections.EMPTY_MAP;
         }
         return new HashMap<>(this.triggers);
     }
 
     Map<String, String> getTriggersAfterSettle() {
         if(this.triggers.isEmpty()) {
+            //noinspection unchecked
             return Collections.EMPTY_MAP;
         }
         return new HashMap<>(this.triggersAfterSettle);
     }
 
     Map<String, String> getTriggersAfterSwap() {
         if(this.triggers.isEmpty()) {
+            //noinspection unchecked
```

Comment:
You can use Collections.emptyMap() and remove the comment

Reply:
Thanks for the tip!